Site/ASAB Maestro/Descriptors/bs-query.yaml

```diff
@@ -0,0 +1,112 @@
+define:
+  type: rc/descriptor
+  name: BS Query
+
+descriptor:
+  image: docker.teskalabs.com/bs-query
+
+  volumes:
+  - "{{SITE}}/{{INSTANCE_ID}}/conf:/conf:ro"
+  - "{{SLOW_STORAGE}}/{{INSTANCE_ID}}:/data/bsquery"
+  # TODO: add jupyter - /data/hdd/jupyter:/data/jupyter
+
+asab:
+  configname: conf/bs-query.conf
+
+  config:
+    storage:
+    # TODO: remove after bs-query has the right folder /data 
+      path: /data/bsquery
+
+    elasticsearch:
+    # TODO: remove after bs-query is ready to read the config section
+      url: "{{ ELASTIC_HOSTS_AUTHORIZED|join(' ') }}"
+
+    auth:
+      multitenancy: "yes"
+      public_keys_url: http://seacat-auth-1:3081/openidconnect/public_keys  # TODO!!!
+
+    pyppeteer:  # TODO !!!
+      url: http://asab-pyppeteer-1:8895
+      username: bs-query
+      password: bs-query
+
+
+seacat-auth:
+  config:
+    "seacatauth:credentials:m2m:machine":
+      mongodb_uri: "{{MONGO_URI}}"
+      mongodb_database: auth
+  content:
+    - "mc.json": |
+        [{
+          "_id": "5f362cf58dca6ae0ad997bfc",
+          "username": "bs-query",
+          "__password": "$2b$12$mKgt17p7sSjHVQO3O00bNO3L3iRcT5MSx06.tQRY7nXL.DtA8rkaq",
```

Comment:
Passwords?

Reply:
I'll make a tech in remote control for this - to create and store the password. I don't like it, but it is the only solution I can think of.